src/components/AppBar.tsx

```diff
@@ -151,9 +151,15 @@ const _AppBar = ({ location }: RouteComponentProps) => {
     ? TESTNET_LAUNCHPAD_URL
     : MAINNET_LAUNCHPAD_URL;
 
-  const networkName = IS_MAINNET
-    ? 'mainnet'
-    : `${EL_TESTNET_NAME.toLowerCase()} testnet`;
+  const networkName = IS_MAINNET ? (
```

Comment:
Any particular reason we create a variable here vs. placing this inline with the HTML like we do elsewhere?